drivers/adodb-firebird.inc.php

```diff
@@ -610,105 +816,106 @@ function UpdateBlobFile($table,$column,$path,$where,$blobtype='BLOB')
 	{
-	$blob_id = fbird_blob_create($this->_connectionID);
+		$blob_id = fbird_blob_create($this->_connectionID);
 
-	// fbird_blob_add($blob_id, $val);
+		// fbird_blob_add($blob_id, $val);
 
-	// replacement that solves the problem by which only the first modulus 64K /
-	// of $val are stored at the blob field ////////////////////////////////////
-	// Thx Abel Berenstein  aberenstein#afip.gov.ar
-	$len = strlen($val);
-	$chunk_size = 32768;
-	$tail_size = $len % $chunk_size;
-	$n_chunks = ($len - $tail_size) / $chunk_size;
+		// replacement that solves the problem by which only the first modulus 64K /
+		// of $val are stored at the blob field ////////////////////////////////////
+		// Thx Abel Berenstein  aberenstein#afip.gov.ar
+		$len = strlen($val);
+		$chunk_size = 32768;
+		$tail_size = $len % $chunk_size;
+		$n_chunks = ($len - $tail_size) / $chunk_size;
 
-	for ($n = 0; $n < $n_chunks; $n++) {
-		$start = $n * $chunk_size;
-		$data = substr($val, $start, $chunk_size);
-		fbird_blob_add($blob_id, $data);
-	}
+		for ($n = 0; $n < $n_chunks; $n++) {
+			$start = $n * $chunk_size;
+			$data = substr($val, $start, $chunk_size);
+			fbird_blob_add($blob_id, $data);
+		}
 
-	if ($tail_size) {
-		$start = $n_chunks * $chunk_size;
-		$data = substr($val, $start, $tail_size);
-		fbird_blob_add($blob_id, $data);
-	}
-	// end replacement /////////////////////////////////////////////////////////
+		if ($tail_size) {
+			$start = $n_chunks * $chunk_size;
+			$data = substr($val, $start, $tail_size);
+			fbird_blob_add($blob_id, $data);
+		}
+		// end replacement /////////////////////////////////////////////////////////
 
-	$blob_id_str = fbird_blob_close($blob_id);
+		$blob_id_str = fbird_blob_close($blob_id);
 
-	return $this->Execute("UPDATE $table SET $column=(?) WHERE $where",array($blob_id_str)) != false;
+		return $this->execute("UPDATE $table SET $column=(?) WHERE $where",array($blob_id_str)) != false;
 
 	}
 
 
-	function OldUpdateBlob($table,$column,$val,$where,$blobtype='BLOB')
-	{
-		$blob_id = fbird_blob_create($this->_connectionID);
-		fbird_blob_add($blob_id, $val);
-		$blob_id_str = fbird_blob_close($blob_id);
-		return $this->Execute("UPDATE $table SET $column=(?) WHERE $where",array($blob_id_str)) != false;
-	}
-
-	// Format date column in sql string given an input format that understands Y M D
-	// Only since Interbase 6.0 - uses EXTRACT
-	// problem - does not zero-fill the day and month yet
-	function SQLDate($fmt, $col=false)
+	/**
+	* Returns a portably-formatted date string from a timestamp database column.
+	*
+	* @link https://adodb.org/dokuwiki/doku.php?id=v5:reference:connection:sqldate
+	*
+	* Firebird does not support an AM/PM format, so the A indicator always shows AM
+	*
+	* @param string $fmt The date format to use.
+	* @param string|bool $col (Optional) The table column to date format, or if false, use NOW().
+	*
+	* @return bool|string The SQL DATE_FORMAT() string, or false if the provided date format was empty.
```

Comment:
Unless I missed something, the function never returns _false_

Reply:
The docblock for this is misleading for this method is misleading in both the prototype and a number of the drivers